releasenotes/notes/oracle-enrich-sqltext-aa778da34a8eb4ed.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Improve SQL text collection.
```

Comment:
Should we add more details here? You are querying a new table now, so maybe that's worth mentioning? Is it possible this change could cause performance impact compared to the old agent code, despite the fact that you are accessing with a `FIXED INDEX`?

Reply:
I measured the performance in both cases under the full orders app load. It's the same ~200ms per query execution.

Reply:
I also put more information in the release notes.

Reply:
> I measured the performance in both cases under the full orders app load. It's the same ~200ms per query execution.

nice! 🚀